src/questions/questions.service.ts

```diff
@@ -252,6 +265,8 @@ export class QuestionsService {
       updated_at: question.updatedAt.getTime(),
       is_follow: hasFollowed,
       is_like: false, // TODO: Implement this.
+      is_answered: is_answered,
+      my_answer_id: answer_id,
```

Comment:
If your `is_answered` shares the same name with the object field, then you can shorten `is_answered: is_answered` to `is_answered`.